### DISCUSSION TITLE
Action button size and coloring

### DISCUSSION THREAD

**Discussion by JesperKawuma** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#33] [Ideas]
Hello @BatoolMM, I have checked the community website and I think the buttons (Join Us and All Materials) should have the same width no matter the content they consume. I also think they should be easily differentiated even on hover. Can I make a pull request about this?

**Comment by JesperKawuma** [comment]
This is what I mean
[Screencast from 10-13-2022 04:28:02 AM.webm](https://user-images.githubusercontent.com/115392523/195477934-baef7ac6-abe3-4fd6-b1af-61a6f986cd5f.webm)

**Comment by BatoolMM** [comment]
Thank you @JesperKawuma, we are reviewing your PR!
